MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,8 +1,131 @@
+    static var ypGreen: UIColor { UIColor(named: "YP Green")! }
+}
+
 final class MovieQuizViewController: UIViewController {
     // MARK: - Lifecycle
+    struct ViewModel {
+        let image: UIImage
+        let question: String
+        let questionNumber: String
+    }
+    // для состояния "Вопрос задан"
+    struct QuizeStepViewModel {
+        let image: UIImage
+        let question: String
+        let questionNumber: String
+    }
+
+    // для состояния "Результат квиза"
+    struct QuizeResultsViewModel {
+        let title: String
+        let text: String
+        let buttonText: String
+    }
+    
+    struct QuizeQuestion {
+        let image: String
+        let text: String
+        let correctAnswer: Bool
+    }
+    private var currentQuestionIndex: Int = 0
+    private var rigthAnswerCount: Int = 0
+    
+    private let questions: [QuizeQuestion] = [
+    QuizeQuestion(image: "The Godfather", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+    QuizeQuestion(image: "The Dark Knight", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+    QuizeQuestion(image: "Kill Bill", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+    QuizeQuestion(image: "The Avengers", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+    QuizeQuestion(image: "Deadpool", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+    QuizeQuestion(image: "The Green Knight", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
+    QuizeQuestion(image: "Old", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false),
+    QuizeQuestion(image: "The Ice Age Adventures of Buck Wild", text: "Рейтинг этого фильма больше чем 6?",correctAnswer: false),
+    QuizeQuestion(image: "Tesla", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false),
+    QuizeQuestion(image: "Vivarium", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false)
+    ]
+    
+    @IBOutlet private weak var imageView: UIImageView!
+    @IBOutlet private weak var textlabel: UILabel!
+    @IBOutlet private weak var counterLabel: UILabel!
+    
+    private func showStep(quize step: QuizeStepViewModel) {
+        imageView.image = step.image
+        textlabel.text = step.question
+        counterLabel.text = step.questionNumber
+    }
+    
+    private func showResult(quize result: QuizeResultsViewModel) {
+        let alert = UIAlertController(
+            title: result.title,
+            message: result.text,
+            preferredStyle: .alert
+        )
+
+        let action = UIAlertAction(title: result.buttonText, style: .default, handler: { _ in
+          print("OK button is clicked!")
+            self.currentQuestionIndex = 0
+            self.rigthAnswerCount = 0
+            self.showStep(quize: self.convert(model: self.questions[0]))
+        })
+
+        alert.addAction(action)
+
+        self.present(alert, animated: true, completion: nil)
```

Comment:
```suggestion
        present(alert, animated: true, completion: nil)
```

swift лаконичный язык, когда можно использовать конструкцию проще - следует использовать ее. В данном случае self избыточен.